src/marqo/core/search/recommender.py

```diff
@@ -202,8 +202,11 @@ def _get_default_interpolation_method(self, marqo_index: MarqoIndex) -> Interpol
         else:
             return InterpolationMethod.LERP
 
-    def _get_exclusion_filter(self, documents: List[str], user_filter: Optional[str]) -> str:
-        not_in = 'NOT (' + ' OR '.join([f'_id:({doc})' for doc in documents]) + ')'
+    def _get_exclusion_filter(self, marqo_index: MarqoIndex, documents: List[str], user_filter: Optional[str]) -> str:
```

Comment:
Do we need a small test on this to track the divergence between structured and unstructured indexes?